### PR TITLE
Add menus for class inheritance to Python plugin.

### PR DESCRIPTION
**Pull request by Osmose**

### CODE REVIEW

**Comment by Osmose** [comment]
@erikrose r?

File: dxr/plugins/clang/menus.py [hunk 1]
```diff
@@ -5,10 +5,10 @@
 from flask import url_for
 
 from dxr.app import DXR_BLUEPRINT
+from dxr.utils import search_url as search
```
Comment (erikrose) [inline dxr/plugins/clang/menus.py]: Good rename.

File: dxr/plugins/python/indexers.py [hunk 2]
```diff
@@ -194,12 +187,23 @@ def visit_ClassDef(self, node):
                               name=name, qualname=qualname,
                               start=start, end=end)
 
+        # Show a menu when hovering over this class.
+        self.yield_ref(start, end,
+                       menus.class_menu(self.file_to_index.tree, class_name))
```
Comment (erikrose) [inline dxr/plugins/python/indexers.py]: It seems like either `class_menu` or `menus.class` (ignoring the keyword for now) would be sufficient to expose your intent. I'd recommend either importing deeper (my favorite) or shortening the routine names.

**Comment by erikrose** [comment]
Aside from that one naming nitpick, r+. Looking good.

**Pull request merged**